enterprise/suppress-user/handler_test.go

```diff
@@ -42,7 +45,7 @@ func (*fakeSuppresser) Suppressed(_, _, _ string) (*model.Metadata, error) {
 	if rand.Intn(2)%2 == 0 { // skipcq: GSC-G404
```

Comment:
We need to remove, `rand.Seed(time.Now().UnixNano())` since it's deprecated.
```suggestion
	if rand.New(rand.NewSource(time.Now().UnixNano())).Intn(2)%2 == 0 { // skipcq: GSC-G404
```